### ISSUE TITLE
Ad for pro version in latest version

### ISSUE THREAD

**Issue by genox** [RSC-Labs/medusa-store-analytics#11]
Hi,

Please reconsider the ad link in the dashboard. It's a huge turn-off. 

Why not set up a project on https://opencollective.com/opensource instead for all the plugins that RSC-labs offers? I would donate instantly. But managing licensing and all the hassle it brings with it is a no go for me.

Edit:

Maybe I need to expand a bit on that, I don't want you to read this as a "hostile" message. The way you solved this right now basically makes the analytics dashboard feel like unregistered shareware. An inferior product compared to what would be available. It's the Wordpress plugin thing. No longer FOSS, but some kind of marketing funnel to nudge people. It's purely psychological and I can only speak for myself.

On the other hand, if there would be a non intrusive, dismissible banner at the bottom of the page with a link to open collective or PayPal and a note about the availability of custom development, that would be completely ok for me and I would immediately consider donating an amount anytime I deploy your plugins in a new client instance. This would make it look much less like Wordpress and much more like a FOSS based approach to financing. This could include setting up fundraisers for new plugins or features of existing ones.

A donation link would also be handy since I wanted to ask you anyways whether I can offer a round of beers since you helped me with setting up the other plugin yesterday. ;)

**Comment by radoslaw-sz** [comment]
hi @genox , thank you for this feedback!
Let me maybe give some insights about this.
Honestly, I still explore possibilities how to manage it commercially. To be fair, I see big potential in Medusa project and that's why I invest my time on this.
I feel bad that you found this "Pro version" as the intrusive or you feel that plugin is shareware version - this was not my intention here for sure. I would rather wanted to emphasize that I can invest much more my time and resources to develop this product if there is commercial interest. 
This plugin will be always under MIT license and nothing will change in this manner. Commercial version will be distributed totally separated with the whole licensing thing and so on. What is more, I have still a big roadmap for MIT, so it is not like I am dropping open source :)
There are plenty of open source projects which have their custom, commercial version with much more features and professional maintenance and my intention is to follow similar approach here.
Sponsorship via open collective or even via Github (like Medusa) is the option which maybe I will consider, but I do not want to rely on this as a primary thing. However, I really appreciate your nice words that you would consider donating <3

To sum up - I do not want (yet) drop an idea about commercial version, but as I mentioned - I am still exploring options (maybe fundraising would be a good option).

Regarding this ad for pro - actually, I was thinking about making this tab "hideable", but that time I did not have a good idea how to manage that. Now, I am thinking that I could approach in two ways:
- hide this Pro Version tab with environment variable set
- add button to hide Pro version tab and use local storage to remember that (it will work if someone will open it again in the same browser).

Do you think that option to hide this Pro Version could be acceptable? Or you still feel that it is not a place for such thing at all?

**Comment by radoslaw-sz** [comment]
hi @genox  - in version 0.12.1 I have added ability to hide this Pro version if someone feels it is intrusive - you can check the README update also: https://github.com/RSC-Labs/medusa-store-analytics?tab=readme-ov-file#hide-pro-version-tab.

I hope you find it as good compromise. Kindly re-open this case if feel differently.
